src/skin.cpp

```diff
@@ -49,6 +49,29 @@ Polygons SkinInfillAreaComputation::getInsidePolygons(const SliceLayerPart& part
  * This function is executed in a parallel region based on layer_nr.
  * When modifying make sure any changes does not introduce data races.
  *
+ * this function may only read/write the skin and infill from the *current* layer.
+ */
+Polygons SkinInfillAreaComputation::getOutlines(const SliceLayerPart& part_here, int layer2_nr)
```

Comment:
It was added accidentally. It's going to be used in CURA-3940, though. Removing it from this PR is only going to give more merge conflicts...

Was it too messy to separate this PR from CURA-3940 ?
Maybe that wasn't such a good idea after all.